docs/research-branch-policy.md

```diff
@@ -0,0 +1,57 @@
+# Research Branch Policy for InstructLab
+
+This document proposes a research branch policy for InstructLab repositories (sdg, eval, training) in the InstructLab organization.
+
+## Background
+
+Research engineers contributing to InstructLab have expressed the need for the ability to make rapid, experimental changes in a space where they can collaborate and move quickly without working through robust, production-level code review. 
+Software engineers contributing to InstructLab have expressed concern about allowing code into the `main` branch that does not meet the production-level quality standards enforced on main. We need to minimize the time from code merged into main to shipping in a release, and automating confidence in that shipment.
```

Comment:
How does the current proposal "minimizes the time from code merged into main to shipping in a release"?

Reply:
It doesn't actually impact it since the proposal only impacts the process previous t the merge to main. Just trying to fairly reflect the various positions and values.